From 37c6b3ad709734bc188a1cb2e17ca70214b4610b Mon Sep 17 00:00:00 2001
From: Zhenbo Gao <zhenbo.gao@windriver.com>
Date: Wed, 29 Aug 2018 16:11:16 +0800
Subject: [PATCH 176/176] tipc: fix compile error related to diag module

Following error is found during building the image:
----
net/tipc/diag.c:110:1: warning: data definition has no type or storage class
|  module_init(tipc_diag_init);
|  ^~~~~~~~~~~

Import linux/module.h to fix this error.

Signed-off-by: Zhenbo Gao <zhenbo.gao@windriver.com>
---
 net/tipc/diag.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/net/tipc/diag.c b/net/tipc/diag.c
index aaabb0b..e717ca0 100644
--- a/net/tipc/diag.c
+++ b/net/tipc/diag.c
@@ -37,6 +37,7 @@
 #include "socket.h"
 #include <linux/sock_diag.h>
 #include <linux/tipc_sockets_diag.h>
+#include <linux/module.h>
 
 static u64 __tipc_diag_gen_cookie(struct sock *sk)
 {
-- 
1.7.9.5

